bazel_example/WORKSPACE.bazel

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A workspace file for the example usage of the gapic-generator-ruby
+"""
+
+workspace(name = "gapic_generator_ruby_example")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+##
+# googleapis for the common protos
+#
+http_archive(
+  name = "com_google_googleapis",
+  strip_prefix = "googleapis-cf41866c6f14f10a07aa1e2a1260fc0a2727d889",
+  urls = ["https://github.com/googleapis/googleapis/archive/cf41866c6f14f10a07aa1e2a1260fc0a2727d889.zip"],
+)
+load("@com_google_googleapis//:repository_rules.bzl", "switched_rules_by_language")
+switched_rules_by_language(name = "com_google_googleapis_imports", gapic = True)
+
+##
+# protobuf
+#
+http_archive(
+  name = "com_google_protobuf",
+  sha256 = "e5265d552e12c1f39c72842fa91d84941726026fa056d914ea6a25cd58d7bbf8",
+  urls = ["https://github.com/protocolbuffers/protobuf/archive/v3.12.3.zip"],
+  strip_prefix = "protobuf-3.12.3",
+)
+load("@com_google_protobuf//:protobuf_deps.bzl", "protobuf_deps")
+protobuf_deps()
+
+##
+# grpc
+#
+http_archive(
+  name = "com_github_grpc_grpc",
+  strip_prefix = "grpc-1.29.1",
+  sha256 = "2afd3e20fd1d52d3d1a605a74befcdcb048a9213a4903880d9267856b063ae60",
+  urls = ["https://github.com/grpc/grpc/archive/v1.29.1.zip"],
+)
+load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
+grpc_deps()
+load("@com_github_grpc_grpc//bazel:grpc_extra_deps.bzl", "grpc_extra_deps")
+grpc_extra_deps()
+
+##
+# rules_proto
+#
+http_archive(
+    name = "rules_proto",
+    sha256 = "602e7161d9195e50246177e7c55b2f39950a9cf7366f74ed5f22fd45750cd208",
+    strip_prefix = "rules_proto-97d8af4dc474595af3900dd85cb3a29ad28cc313",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz",
+        "https://github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz",
+    ],
+)
+load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
+rules_proto_dependencies()
+rules_proto_toolchains()
+
+##
+# gapic_generator_ruby
+# (either local or from github)
+#
+
+# use the following to get a consistent external version from github
+# http_archive(
+#     name = "gapic_generator_ruby",
+#     urls = ["https://github.com/viacheslav-rostovtsev/gapic-generator-ruby/archive/bea87e2ec0459310e56b3db3c3198d8accf2ff4b.zip"],
```

Comment:
Do you want to use a link to the origin repo rather than your fork, in this example?